sample/src/androidTest/java/com/davidecirillo/multichoicesample/SampleToolbarActivityTest.java

```diff
@@ -147,7 +147,7 @@ public void testMultiChoiceToolbar() {
                 .perform(RecyclerViewActions.actionOnItemAtPosition(0, longClick()));
 
         onView(allOf(isAssignableFrom(TextView.class), withParent(isAssignableFrom(Toolbar.class))))
-                .check(matches(withText(containsString("1"))));
+                .check(matches(withText(containsString("One"))));
```

Comment:
You shouldn't change this test because this was checking that the item number has changed to the correct, create some new test, with the specific plural scenarios about the feature that you added and make sure al the tests instrumental and unit are passing!